modules/connectors/ninja_van/karrio/providers/ninja_van/utils.py

```diff
@@ -0,0 +1,20 @@
+
+import karrio.core as core
+
+
+class Settings(core.Settings):
+    """Ninja Van connection settings."""
+
+    # username: str  # carrier specific api credential key
+
+    @property
+    def carrier_name(self):
+        return "ninja_van"
+
+    @property
+    def server_url(self):
+        return (
+            "https://api.ninjavan.co"
+            if self.test_mode
+            else "https://sandbox.carrier.api"
+        )
```

Comment:
This was closed. Just the opposite:

```suggestion
    def server_url(self):
        return (
            "https://api-sandbox.ninjavan.co"
            if self.test_mode
            else "https://api.ninjavan.co"
        )
```